adrian/script.js

```diff
@@ -0,0 +1,60 @@
+//Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+let integer0 = 55;
+let integer1 = 86;
+if ((integer0 >= 50 && integer0 <= 99) || (integer1 >= 50 && integer1 <= 99)) {
+  console.log(true);
+}
+
+//Check if three given integer values are in the range 50 to 99 (inclusive). Print true if one or more of them are in the range.
+let integer2 = 44;
+if (
+  (integer0 >= 50 && integer0 <= 99) ||
+  (integer1 >= 50 && integer1 <= 99) ||
+  (integer2 >= 50 && integer2 <= 99)
+) {
+  console.log(true);
+}
+// Declare the variables a, b and c, and give them number values.
+//Check which one out of the three variables has the largest value and print it to the console.
+//Then change the values of the variables to see if your conditional still works.
+let a = 0;
+let b = 1;
+let c = 2;
+//c = 34
+console.log(Math.max(a, b, c));
+// Create a new string adding "Py" in front of a given string. If the given string begins with "Py" then print the original string.
+let givenString = "hello";
+let newString = `Py${givenString}`;
+let originalString = "The original string?";
+if (givenString.slice(0, 2) === "Py") console.log(originalString);
+// Calculate the sum of the two integers. If the sum is in the range 50 to 80, print 65, otherwise print 80.
+let integer3 = 34;
+let integer4 = 17;
+if (integer3 + integer4 > 50 && integer3 + integer4 < 70) console.log(65);
```

Comment:
In this condition the range is between 50.000000000001 and 69.9999999 and not between 50 and 80

Reply:
Done, I did it now to include 50 and 80